controller/templates/controller/partials/app_tree.html

```diff
@@ -0,0 +1,11 @@
+<sl-tree class="custom-icons">
+<sl-icon name="plus-square" slot="expand-icon"></sl-icon>
+<sl-icon name="dash-square" slot="collapse-icon"></sl-icon>
+{{ tree }}
+</sl-tree>
+<style>
+        .custom-icons sl-tree-item::part(expand-button) {
+          /* Disable the expand/collapse animation */
```

Comment:
What do you have against the expand/collapse animation?

Reply:
I don't like them, so I guess just an opinionated choice. Can change it if there's appetite for animations.